test/CappedCrowdsale.js

```diff
@@ -62,7 +70,7 @@ contract('CappedCrowdsale', function ([_, wallet]) {
   describe('ending', function () {
 
     beforeEach(async function () {
-      await advanceToBlock(this.startBlock - 1)
+      await increaseTime(moment.duration(1, 'week'))
```

Comment:
I don't like that the time duration until the crowdsale starts is hardcoded here. I think we should define a function analogous to `advanceToBlock` but for time, that internally calculates how many seconds it is necessary to increase. Suppose we call it `increaseTimeTo`, that we can use it like `increaseTimeTo(this.startTime)`.

It should be done like that throughout all tests in this PR.